### PR TITLE
Internal: Add support for tag to declare it supports atomic widgets [EDS-1065]

### PR DESCRIPTION
**Pull request by Omerisra6**

### CODE REVIEW

File: modules/atomic-widgets/dynamic-tags/dynamic-tags-registry.php [hunk 1]
```diff
@@ -74,19 +74,16 @@ private function convert_dynamic_tag_to_atomic( $tag ) {
...
 
-			$atomic_dynamic_tag['atomic_controls'] = $controls;
-			$atomic_dynamic_tag['props_schema'] = $props_schema;
-		} catch ( \Exception $e ) {
+		if ( ! $tag_atomic_schema ) {
 			return null;
 		}
 
-		return $atomic_dynamic_tag;
+		return array_merge( $atomic_dynamic_tag, $tag_atomic_schema );
```
Comment (blackcolt) [inline modules/atomic-widgets/dynamic-tags/dynamic-tags-registry.php:83]: return array_merge( $atomic_dynamic_tag, $atomic_tag_schema );

File: modules/atomic-widgets/dynamic-tags/dynamic-tags-registry.php [hunk 2]
```diff
@@ -125,14 +132,18 @@ private function convert_control_to_atomic( $control ) {
...
 		if ( ! isset( $control['name'], $control['section'], $control['label'], $control['default'] ) ) {
-			throw new \Exception( 'Control must have name, section, label and default' );
+			return null;
 		}
 
-		return $map[ $control['type'] ]( $control );
+		try {
+			return $map[ $control['type'] ]( $control );
+		} catch ( \Exception $e ) {
+			return null;
```
Comment (blackcolt) [inline modules/atomic-widgets/dynamic-tags/dynamic-tags-registry.php:145]: Don't we want to log it?

File: modules/atomic-widgets/dynamic-tags/dynamic-tags-registry.php [hunk 3]
```diff
@@ -74,19 +74,16 @@ private function convert_dynamic_tag_to_atomic( $tag ) {
 			return $atomic_dynamic_tag;
 		}
 
-		try {
-			['atomic_controls' => $controls, 'props_schema' => $props_schema] = $this->convert_controls_to_atomic( $tag['controls'] );
+		$tag_atomic_schema = $this->convert_controls_to_atomic( $tag['controls'], $tag['force_convert_to_atomic'] ?? false );
```
Comment (blackcolt) [inline modules/atomic-widgets/dynamic-tags/dynamic-tags-registry.php:77]: if the default is false, why to pass it?

File: tests/phpunit/elementor/modules/atomic-widgets/dynamic-tags/test-dynamic-tags-module.php [hunk 4]
```diff
@@ -435,6 +482,12 @@ public function cleanup() {
 				Plugin::$instance->controls_manager->delete_stack( $this );
 			}
 
+			public function get_editor_config() {
+				$conf = parent::get_editor_config();
```
Comment (blackcolt) [inline tests/phpunit/elementor/modules/atomic-widgets/dynamic-tags/test-dynamic-tags-module.php:486]:
```suggestion
	return array_merge( $[conf](parent::get_editor_config()), $this->options['editor_config'] ?? [] );
```

**Review by Nevoss — DISMISSED** [review]

**Review by Nevoss — APPROVED** [review]

**Pull request merged**